docs/content/developer/getting-started/simple-token-transfer.mdx

```diff
@@ -0,0 +1,438 @@
+│  │ Digest: AqFmepZJmqEMi2qPYscZf1X4vZtXs3oUgSg2vn2X7CDz                                                                         │
+│  │ Modules: my_token                                                                                                            │
+│  └──                                                                                                                            │
+╰─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
+╭───────────────────────────────────────────────────────────────────────────────────────────────────╮
+│ Balance Changes                                                                                   │
+├───────────────────────────────────────────────────────────────────────────────────────────────────┤
+│  ┌──                                                                                              │
+│  │ Owner: Account Address ( 0xedeca9d6897cf86917e45f5c5cd596da48a5c4e3e1d9425aee42bed37c9ee44d )  │
+│  │ CoinType: 0x2::iota::IOTA                                                                      │
+│  │ Amount: -14444400                                                                              │
+│  └──                                                                                              │
+╰───────────────────────────────────────────────────────────────────────────────────────────────────╯
+```
+
+## Simple Token Transfer UI
+
+Let’s now set up the frontend application using the [IOTA dApp kit](../../references/ts-sdk/dapp-kit/index.mdx). This kit provides a collection of tools and components that simplify integrating IOTA-based smart contracts into your web applications. By leveraging the dApp kit, you can efficiently build user interfaces that enable users to interact with the blockchain, such as sending transactions, querying contracts, and displaying data.
+
+The dApp kit is designed to streamline the connection between your frontend and the IOTA blockchain, handling important tasks like wallet integration, transaction signing, and communication with your deployed contract. If you’re new to the dApp kit, you can consult its documentation for comprehensive instructions on how to set it up and use its features to create a smooth frontend experience.
+
+In this section, we’ll walk you through the process of configuring the frontend and linking it to the smart contract we created earlier. By the end, you’ll have a fully functional dApp that enables users to interact with your IOTA-based token transfer contract directly from the frontend.
+
+### Setup 
+
+Run the following the command to setup the initial IOTA dApp.
+
+```bash
+pnpm add @iota/dapp-kit @iota/iota-sdk @tanstack/react-query
+```
+
+### Code Structure
+
+- Components: Contains the components listed below for the UI :
```

Comment:
This sentence isn't very clear, could you expand it a bit?

Reply:
Sure, I will update it in the next commit.